src/encoded/static/components/award.js

```diff
@@ -543,90 +1498,621 @@ ChartRenderer.propTypes = {
+const ExperimentDate = (props) => {
+    const { experiments, award } = props;
+    let dateReleasedArray = [];
+    let dateSubmittedArray = [];
+    const deduplicatedreleased = {};
+    const deduplicatedsubmitted = {};
+    let label = [];
+    let data = [];
+    let cumulativedatasetReleased = [];
+    let cumulativedatasetSubmitted = [];
+    let accumulatorreleased = 0;
+    let accumulatorsubmitted = 0;
+    let monthreleaseddiff = 0;
+    let monthsubmitteddiff = 0;
+    const fillreleasedDates = [];
+    const fillsubmittedDates = [];
+
+    // Search experiments for month_released and date_submitted in facets
+    if (experiments && experiments.facets && experiments.facets.length) {
+        const monthReleasedFacet = experiments.facets.find(facet => facet.field === 'month_released');
+        const dateSubmittedFacet = experiments.facets.find(facet => facet.field === 'date_submitted');
+        dateReleasedArray = (monthReleasedFacet && monthReleasedFacet.terms && monthReleasedFacet.terms.length) ? monthReleasedFacet.terms : [];
+        dateSubmittedArray = (dateSubmittedFacet && dateSubmittedFacet.terms && dateSubmittedFacet.terms.length) ? dateSubmittedFacet.terms : [];
+    }
+
+    // Use Moment to format arrays of submitted and released date
+    const standardreleasedTerms = dateReleasedArray.map((term) => {
+        const standardDate = moment(term.key, ['MMMM, YYYY', 'YYYY-MM']).format('YYYY-MM');
+        return { key: standardDate, doc_count: term.doc_count };
+    });
 
-const Award = (props) => {
-    const { context } = props;
-    const statuses = [{ status: context.status, title: 'Status' }];
+    const standardsubmittedTerms = dateSubmittedArray.map((term) => {
+        const standardDate = moment(term.key, ['MMMM, YYYY', 'YYYY-MM']).format('YYYY-MM');
+        return { key: standardDate, doc_count: term.doc_count };
+    });
 
-    return (
-        <div className={globals.itemClass(context, 'view-item')}>
-            <header className="row">
-                <div className="col-sm-12">
-                    <h2>{context.title || context.name}</h2>
-                    <div className="status-line">
-                        <div className="characterization-status-labels">
-                            <StatusLabel status={statuses} />
-                        </div>
-                    </div>
-                </div>
-            </header>
+    // Sort arrays chronologically
+    const sortedreleasedTerms = standardreleasedTerms.sort((termA, termB) => {
+        if (termA.key < termB.key) {
+            return -1;
+        } else if (termB.key < termA.key) {
+            return 1;
+        }
+        return 0;
+    });
+    const sortedsubmittedTerms = standardsubmittedTerms.sort((termA, termB) => {
+        if (termA.key < termB.key) {
+            return -1;
+        } else if (termB.key < termA.key) {
+            return 1;
+        }
+        return 0;
+    });
 
```

Comment:
Same thing here. These two sorts do the same thing, and can be implemented just once as as function.